README.rst

```diff
@@ -6,7 +6,7 @@ This is an adapter that accepts Prometheus remote read/write requests,
 and sends them on to CrateDB. This allows using CrateDB as long term storage
 for Prometheus.
 
-Requires CrateDB 2.2.0 or greater.
+Requires CrateDB 3.1.0 or greater.
```

Comment:
Btw.,  CrateDB 3.1.0 is not out yet, but my nightly build that has the required features says `3.1.0-SNAPSHOT` on startup, so I assume it'll be 3.1.0 once it's released.

Reply:
Yes, the BEGIN/COMMIT feature + the fix for the TIMESTAMPTZ type will be available in CrateDB 3.1.0.